packages/amqp-client-async/amqp-client-async.2.2.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "Anders Fugmann <anders@fugmann.net>"
+authors: [ "Anders Fugmann" ]
+homepage: "https://github.com/andersfugmann/amqp-client"
+bug-reports: "https://github.com/andersfugmann/amqp-client/issues"
+dev-repo: "git+https://github.com/andersfugmann/amqp-client.git"
+doc: "https://andersfugmann.github.io/amqp-client/amqp-client-async/Amqp_client_async/"
+license: "BSD3"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.1"}
```
Dune should not be tagged as a build dependency anymore (see https://github.com/ocaml/opam-repository/pull/14266). Also the `dune-project` at the root of your source file says you require at least dune 1.1.